components/library_hero.vue

```diff
@@ -53,18 +59,38 @@
     import Star from '@fortawesome/fontawesome-free/svgs/solid/star.svg?inline';
     import GitHub from '@fortawesome/fontawesome-free/svgs/brands/github.svg?inline';
     import Npm from '@fortawesome/fontawesome-free/svgs/brands/npm.svg?inline';
+    import ShieldAlt from '@fortawesome/fontawesome-free/svgs/solid/shield-alt.svg?inline';
     import formatUnits from '../util/format_units';
     import utm from '../util/utm';
+    import getVulns from '../util/get_vulns';
 
     export default {
         name: 'LibraryHero',
         components: {
             Star,
             GitHub,
             Npm,
+            ShieldAlt,
         },
         props: {
             library: Object,
+            version: String,
+        },
+        data () {
+            return {
+                vulns: null,
+            };
+        },
+        watch: {
+            version: {
+                immediate: true,
+                async handler () {
+                    this.vulns = await getVulns(this.$props.library, this.$props.version);
```

Comment:
```suggestion
                    this.vulns = await getVulns(this.$props.library, this.$props.version).catch(() => {});
```

This call failing will cause Nuxt to show the error page -- we can just add a .catch() on the end, so it is undefined it it fails?